src/app/timer/timer.component.html

```diff
@@ -1 +1,11 @@
-<div class="timer-container">timer works!</div>
+<div class="timer-label">Time until end of match</div>
```

Comment:
that label makes no sense if there are no games to start, or game is finished or "this game will start soon"

Reply:
changed